### ISSUE TITLE
Cluster creation does not provide enough feedback for the current status

### ISSUE THREAD

**Issue by cnotv** [rancher/dashboard#5398]
**Detailed Description**

While creating a cluster, we simply display a `Provisioning` label, which does not make it clear if:

- Something is actually still happening
- The cluster has been created
- Amount of time required (approximately)
- Logs of the ongoing steps are quite hidden

**Context**

It would be useful to inform the user that the functionality is not broken, rather it has been actually executed correctly. To cover the typical kind of information we ask in the chat, would be rather convenient to have them explicitly defined.

**Comment by cnotv** [comment]
Another user in the public chat complained of the same issue I keep facing while using Rancher:
https://rancher-users.slack.com/archives/C3ASABBD1/p1655820308486799

<img width="1400" alt="Screenshot 2022-06-24 at 11 12 35" src="https://user-images.githubusercontent.com/5009481/175504560-b14aa3ef-5083-4a71-9ab3-dfc26378e47e.png">

**Comment by kwwii** [comment]
- Is there a way to calculate the expected time of this process as a whole?
 - Is there a way to calculate the length of specific parts? Which ones are indefinable?
 - Can we somehow get more detailed information than "Pending", "Provisioned", and "Ready"?
 - Are there criteria to determine, once a cluster is "Ready", if is has reached a further state? (Provisioned? Has-some-kind-of-configuration, etc?)
 - Can we learn/assume something by the length of any steps based on the length of previous steps?
 - Can we set maximum time limits for any steps? (for example: "after X minutes show something is wrong with the current step")

**Comment by catherineluse** [comment]
The usual way to troubleshoot this kind of issue is to get the logs in the Rancher agent. Maybe the UI could make it easier to get the logs by giving you a premade command to run on the cluster node.

**Comment by kwwii** [comment]
Can we add a link to "View logs"?

**Comment by cnotv** [comment]
Yeah we have a tab which uses anchors, so we may have `<domain>/c/_/manager/provisioning.cattle.io.cluster/<cluster-name>#log`.

**Comment by nwmac** [comment]
Going to close as Stale. Please re-open if you feel strongly about this feature.
